extensions/elytron-security-oauth2/runtime/src/main/java/io/quarkus/elytron/security/oauth2/runtime/OAuth2Config.java

```diff
@@ -0,0 +1,49 @@
+package io.quarkus.elytron.security.oauth2.runtime;
+
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+/**
+ * See http://docs.wildfly.org/14/WildFly_Elytron_Security.html#validating-oauth2-bearer-tokens
+ */
+@ConfigRoot(name = "oauth2", phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
+public class OAuth2Config {
+    /**
+     * If the embedded store is enabled.
+     */
+    @ConfigItem(defaultValue = "true")
+    public boolean enabled = true;
```

Comment:
the comment seems to refer to an embedded store, whereas the property makes me think it enables OAuth2 support entirely.

Reply:
done